### PR TITLE
fix: injected sql model now uses chart name instead of random ID for improved error message

### PR DESCRIPTION
**Pull request by louisjoecodes**
- Instead of using a random ID for the injected SQL models, just use the Chart file name
- This is guaranteed to be unique as 2 charts should not have the same name

### CODE REVIEW

File: rust/core/src/project_to_sql.rs [hunk 1]
```diff
@@ -23,13 +23,12 @@ pub async fn project_and_fs_to_query_sql_for_model_sql(
     project_root: &str,
```
Comment (benfdking) [inline rust/core/src/project_to_sql.rs:23]: nit: explaining the purpose of the model_name might be useful.

File: rust/core/src/project_to_sql.rs [hunk 2]
```diff
@@ -1475,6 +1467,7 @@ sources:
             project_root,
             model,
             None,
+            "shift_hours.chart.sql",
```
Comment (benfdking) [inline rust/core/src/project_to_sql.rs:1470]: nit: This doesn't quite align with the expected behaviour. 

nit: Might be worth putting in a fake test and validating the error message to make it's something we count on.

File: proto/quary/service/v1/wasm_rust_rpc_calls.proto [hunk 3]
```diff
@@ -312,6 +312,7 @@ message ReturnDefinitionLocationsForSQLResponse {
 message ReturnSQLForInjectedModelRequest {
   string project_root = 1;
   string sql = 2;
+  string model_name = 3;
```
Comment (benfdking) [inline proto/quary/service/v1/wasm_rust_rpc_calls.proto:315]: nit: explaining what this does might be useful.

**Review by benfdking — APPROVED** [review]
Little things that would make the reason for the change in API slightly more explicit.

**Comment by louisjoecodes** [comment]
- Nits resolved
- Throw error (to trigger sentry) if file-name/chart name retrieval fails

**Pull request merged**